simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountAuthenticationViewBindings.kt

```diff
@@ -252,11 +252,11 @@ sealed class AccountAuthenticationViewBindings {
     }
 
     fun getPassword(): AccountPassword {
-      return AccountPassword(this.pass.text.toString())
+      return AccountPassword(this.pass.text.toString().trim())
```

Comment:
Sad to think that this had been the problem for all of this time...

Reply:
Hopefully!